src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -157,8 +129,22 @@ function removeCurrentHeaderFooter() {
   });
 }
 
+const startVCLModal = () => {
+  const vclContainer = document?.getElementById('ts-vcl-container');
+
+  startReactApp(
+    <Provider store={store}>
+      <CrisisLineModal />
+    </Provider>,
+    vclContainer,
+  );
+
+  addFocusBehaviorToCrisisLineModal();
+  addOverlayTriggers();
+};
+
+// Add modernized header and footer
 function activateInjectedAssets() {
-  activateHeaderFooter();
   fetch(`${getContentHostName()}/generated/headerFooter.json`)
```

Comment:
Good question. There is no fallback for this, but it does get a bit hairy because that would be a failure on content-build's part to generate the file in the vets-website repo. Locally, if I delete the `headerFooter.json` file, the header and footer simply don't show up. I'm not sure who would own that research / fix, though. We don't own the nuts and bolts of content-build. @FranECross @jilladams thoughts?

I won't block this PR on this comment (as you've mentioned elsewhere) since it is out of scope for these changes, though.

Reply:
> it does get a bit hairy because that would be a failure on content-build's part to generate the file in the vets-website repo

Ah that's interesting for sure. Which team does own the nuts and bolts of content-build...cms?

Reply:
Yes, the CMS team owns it